src/nvim/tui/terminfo.c

```diff
@@ -20,7 +20,7 @@
 #endif
 
 // One creates the dumps from terminfo.src by using
-//      od -t d1 -w | cut -c9- | sed -e 's/\>/,/g'
+//      od -t d1 -w | cut -c9- | sed -e 's/\>\s*/, /g'
```

Comment:
Why remove the whitespace?

Reply:
To save space? They should be all superfluous because the actual data are integers by then.

Reply:
It also makes it harder to compare changes.

Reply:
Well, I wanted to automate everything with a script anyway, but there are other things to sort out first. E.g. what modifications we actually did to Dickey's terminfo.src before compiling.